MarvelApp/MarvelApp/Network/Endpoints/HomePageEndpoints.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  HomePageEndpoints.swift
+//  MarvelApp
+//
+//  Created by Caner Tüysüz on 5.12.2024.
+//
+
+import Foundation
+
+enum HomepageEndpoints: EndpointProtocol {
+    case getCharacters
+
+    var url: String {
+        switch self {
+        case .getCharacters:
+            return "https://gateway.marvel.com:443/v1/public/characters"
```

Comment:
You can go to more than one service belonging to this base endpoint. Copying and pasting is not correct.

Like:
```suggestion
            protocol EndpointProtocol {
              var baseURL: String { get } // default value can be set
              var path: String { get }
            }
```